uaa/src/main/webapp/WEB-INF/spring/multitenant-endpoints.xml

```diff
@@ -24,10 +24,14 @@
           entry-point-ref="oauthAuthenticationEntryPoint"
           use-expressions="true" authentication-manager-ref="emptyAuthenticationManager"
           xmlns="http://www.springframework.org/schema/security">
-        <intercept-url pattern="/orchestrator/zones" access="#oauth2.hasScopeInAuthZone('zones.{zone.id}.admin') or #oauth2.hasScope('zones.write')" method="POST"/>
-        <intercept-url pattern="/orchestrator/zones" access="#oauth2.hasScopeInAuthZone('zones.read') or #oauth2.hasScopeInAuthZone('zones.{zone.id}.admin') or #oauth2.hasScope('zones.write')" method="GET"/>
-        <intercept-url pattern="/orchestrator/zones" access="#oauth2.hasScopeInAuthZone('zones.{zone.id}.admin') or #oauth2.hasScope('zones.write')" method="DELETE"/>
-        <intercept-url pattern="/orchestrator/zones" access="#oauth2.hasScopeInAuthZone('zones.{zone.id}.admin') or #oauth2.hasScope('zones.write')" method="PUT"/>
+        <intercept-url pattern="/orchestrator/zones"
+                       access="#oauth2.hasScopeInAuthZone('orchestrator.zones.write') or #oauth2.hasScopeInAuthZone('zones.{zone.id}.admin') or #oauth2.hasScope('zones.write')" method="POST"/>
```

Comment:
Can we add all mandatory authorities with 'and' operator? instead of using or. This means that we can call the post API only with 'orchestrator.zones.write' or 'zones.write'.

Reply:
As discussed with the team, orchestrator zone API should be available for orchestrator specific client with orchestrator.zones.* authorities, as well as UAA admin client with zones.* authorities or zone admin clients with  zones.{zone.id}.admin authorities.  ANY of these scopes should give access to orchestrator zone APIs.

That's said, when checking for required scopes, what's the difference between oauth2.hasScopeInAuthZone and oauth2.hasScope? Which one should be used to check orchestrator.zones.* scopes?